src/Standards/PSR12/Docs/ControlStructures/BooleanOperatorPlacementStandard.xml

```diff
@@ -0,0 +1,57 @@
+<documentation title="Boolean Operator Placement">
+    <standard>
+    <![CDATA[
+    Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Boolean operator between conditions at the beginning of the line.">
+        =<![CDATA[
+if (
+    $expr1
+    && $expr2
+    && ($expr3
+    || $expr4)
+    && $expr5
+) {
+    // if body.
+}
+        ]]>
+        </code>
+        <code title="Invalid: Boolean operators between conditions at the beginning and the end of the line.">
+        <![CDATA[
+if (
+    $expr1 &&
+    ($expr2 || $expr3)
+    && $expr4
+) {
+    // if body.
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: Boolean operator between conditions at the end of the line.">
+        <![CDATA[
+if (
+    $expr1 &&
```

Comment:
Maybe ? Just for the variation/showing that the rule applies with a mix of operators, not only when the operators are the same.
```suggestion
    $expr1 ||
```